stamina-core/src/test/scala/stamina/TestOnlyPersister.scala

```diff
@@ -1,17 +1,18 @@
 package stamina
 
-import scala.reflect._
 import akka.actor._
 import akka.serialization._
+import scala.reflect.ClassTag
+import scala.reflect.runtime.universe._
 
 object TestOnlyPersister {
   private val system = ActorSystem("TestOnlyPersister")
   private val javaSerializer = new JavaSerializer(system.asInstanceOf[ExtendedActorSystem])
   import javaSerializer._
 
-  def persister[T <: AnyRef: ClassTag](key: String): Persister[T, V1] = new JavaPersister[T](key)
+  def persister[T <: AnyRef: ClassTag](key: String)(implicit typeTag: TypeTag[T] = null): Persister[T, V1] = new JavaPersister[T](key)(Option(typeTag))
 
-  private class JavaPersister[T <: AnyRef: ClassTag](key: String) extends Persister[T, V1](key) {
+  private class JavaPersister[T <: AnyRef: ClassTag](key: String)(typeTagOption: Option[TypeTag[T]] = None) extends Persister[T, V1](key)(typeTagOption) {
```

Comment:
Hmm, this line now triggers a warning:

```
[warn] /home/aengelen/dev/stamina/stamina-core/src/test/scala/stamina/TestOnlyPersister.scala:15: private object in object TestOnlyPersister is never used
[warn]   private class JavaPersister[T <: AnyRef: ClassTag](key: String)(typeTagOption: Option[TypeTag[T]] = None) extends Persister[T, V1](key)(typeTagOption) {
[warn]                 ^
```

Not sure what that's about, as it definitely appears to be still in use...